app/models/concerns/hydranorth/generic_file/ga_stats.rb

```diff
@@ -0,0 +1,11 @@
+module Hydranorth
+  module GenericFile
+    module GaStats 
```

Comment:
A more descriptive name would make this easier to keep track of in the future (since these stats don't come from GA) - could we call it ERA1Stats, both in the ruby and the javascript.